turnwithus/lytics.com
# LYT-91: Enhance blog pagination with AJAX

* adds fetch and a promise polyfill
* adds script to fetch additional posts and add them to list
* binds/unbinds & hides/shows applicable more buttons
* adds a bit of CSS to help
* adds a couple of new npm scripts to the Craft `package.json` to compile and copy over only JS or CSS fro mthe pattern library, because _faster_ ✨ 

**To test:**
* go to `/blog` in Craft instance (not pattern library)
* click Load More button at bottom of blog items
* new items should load in and be visible on page
* button that was clicked should not be visible
* new Load More button should then be clickable and load in another set of posts, etc.

**To test the fail case:**
* open js/blog/load-more.js
* change line 22 (`if (response.status >= 200 && response.status < 300) {`) to _anything_ that is falsey (ie, 1 > 2) OR just comment out that entire `if` statement (lines 22-24) so `checkStatus` always returns an error state
* repeat steps above
* rather than new posts loading in, an error message should appear below the initial Load More button that was clicked. Current error text: "There has been a problem retrieving more results. Please try again."
